features/coffeescript_compilation_error.feature

```diff
@@ -3,6 +3,12 @@ Feature: Coffeescript compilation error
   As a developer with an error in my coffeescript file
   I want an appropriate error message
 
+  Background:
+    Given I have configured "filePattern" to be "**/*.coffee"
+    And I have configured "transpilers" to contain
+      | extension | module        |
```

Comment:
Had them uppercase to start but since cucumber-js doesn't yet have mapHeaders, this was just easier to implement. Looks like I can just use lodash's mapValues in the interim.